* eDNA literature
- Pawlowski, J., Kelly-Quinn, M., Altermatt/ F., Apotheloz-Perret-Gentil, L., Beja P. et al. (2018)
  The future of biotic indices in the ecogenomic era: Integrating (e)DNA metabarcoding in biological 
  assessment of aquatic ecosystems. Science of the Total Environment, 637-638, 1295-1310.
  [[https://www.sciencedirect.com/science/article/pii/S0048969718316322][Science Direct]]
  
  A review looking at how and if eDNA monitoring can be used as part of biomonitoring. Both problems and promises
  with eDNA has to be taken into consideration. The major issue with the approach as it stands is the lack of standardized
  procedures as well as the lack of knowledge regarding primer biases. This coupled with incomplete reference databases
  makes it very challenging to convert read counts to species and compare with traditional phenotypically based estimation
  of species abundance/diversity. For some organisms groups we are closer to having a working standard. For fish species
  they do for example argue for national efforts to fill in gaps in reference databases and take of efforts to include eDNA
  data in estimates of biological indexes.
  
  An interesting part of the paper discuss the use of machine-learning for estimating indexes from eDNA data. 
  This could be useful as it would in theory make it possible to use all sequence data without converting reads to species. 
  Hence instead of relying on incomplete databases and massive loss of data from the read data we would make use of knowledge
  obtained in certain system and learn about the expected eDNA signals stemming from such a water and algorithmically estimate 
  indexes based on these.

- Thomsen, P.F., Møller, P.R., Sigsgaard, E.E., Knudsen, S.W.,
  Jørgensen, O.A. and Willerslev, E. (2016). Environmental DNA from
  seawater samples correlate with trawl catches of subarctic, deepwater
  fishes. PloS one, 11(11), p.e0165252
  [[http://journals.plos.org/plosone/article?id%3D10.1371/journal.pone.0165252][PloS one]]

  This paper shows a strong agreement between species composition found
  by trawling and eDNA sequencing. They use 12sRNA and despite using a
  blocking primer to prevent human DNA the most common contaminants seen
  were still human.

- Rees, H. C., Maddison, B. C., Middleditch, D. J., Patmore, J. R., &
  Gough, K. C. (2014). The detection of aquatic animal species using
  environmental DNA–a review of eDNA as a survey tool in
  ecology. Journal of Applied Ecology, 51(5), 1450-1459
  [[https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/1365-2664.12306][Journal of Applied Ecology]]

  Among things they discuss the sampling strategy and emphasis that
  taking at least three samples from a water body increases the chance
  of detecting species in the water. Most studies uses between 3-10
  PCR replicates when using q-PCR. Using both positive and negative
  controls is essential for correct conclusions.

  They highlight some differences in cost for detecting invasive species
  
  "In terms of sampling effort, eDNA analysis can have
   considerable time and cost beneﬁts over traditional survey
   methods, especially when looking at the distribution of
   rare or threatened species. In a study of invasive Asian
   carp in Chicago, Illinois, it took 93 days of person effort
   to detect one silver carp by electroﬁshing at a site,
   whereas eDNA analysis required only 0.174 days person
   effort to achieve a positive detection (Jerde et al. 2011)"
   
  In spite of high cost for sequences there are hence money to save by
  using clever sampling strategies and DNA sequencing, compared to
  more traditional monitoring approaches.

- Shaw, J. L., Clarke, L. J., Wedderburn, S. D., Barnes, T. C., Weyrich, L. S., & Cooper, A. (2016). Comparison of
  environmental DNA metabarcoding and conventional fish survey methods in a river system. Biological Conservation, 197, 
  131-138
  [[https://www.sciencedirect.com/science/article/pii/S000632071630088X][Biological Conservation]]
  
  The effect of taking multiple samples from both water and sediment is discussed.
  One of the main conclusion is that the water samples correlates better with netting compared to sediments. In addition
  the detection of DNA from the less common species is more effecient at sample size 5 compared to only 2 samples.
  
- Hinlo, R., Gleeson, D., Lintermans, M., & Furlan, E. (2017). Methods to maximise recovery of environmental DNA from water 
  samples. PloS one, 12(6), e0179251
  [[http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0179251][PLoS ONE]] 
  
  This paper compares both the effects of different approaches of extracting DNA from water. They testing the effect of
  type of filter paper as well as extraction and storage of water after sampling. In short one see differences between types of
  filters. They also note that filtering samples as soon as possible (the same day) or DNA is lost. Especially freezing seems
  to negatively effect the concentration of DNA.
